pkg/symbol/symbol.go

```diff
@@ -90,7 +90,7 @@ func (s *Symbolizer) resolveJITedFunctions(pid profiler.PID, locations []*profil
 		if !ok {
 			sym, err := perfMap.Lookup(loc.Address)
 			if err != nil {
-				if !errors.Is(err, perf.ErrNotFound) {
+				if !errors.Is(err, perf.ErrPerfMapNotFound) {
 					continue
 				}
```

Comment:
This condition is not necessary, `Lookup()` only returns `ErrNoSymbolFound`

```suggestion
```

Reply:
Good catch ! ^ 2